packages/atlas/lib/src/atlas.dart

```diff
@@ -33,6 +33,9 @@ class Atlas extends StatelessWidget {
   /// which can be used to manipulate the map.
   final ArgumentCallback<AtlasController> onMapCreated;
 
+  /// 'onLocationChanged' Callback that receives a position when the user moves
+  final ArgumentCallback<LatLng> onLocationChanged;
```

Comment:
So the map component is responsible for tracking the user's location in real-time?

Reply:
@felangel yes. For example the HERE provider uses a location engine supplied by the HERE SDK. This engine is responsible for receiving updates from the user's location if permission has been given to the client application. If the client application has `showMyLocation` set to true, the mapProvider will intercept this location update and use it to show the user's current location on the map.